### PR TITLE
Updates to Kata Containers tutorial

### PR DESCRIPTION
**Pull request by intelkevinputnam**
1. Added note for proxy config for docker
2. Improved wording throughout
3. Removed asterisks where they weren't needed
4. Removed reference to version number

### CODE REVIEW

File: source/clear-linux/tutorials/kata.rst [hunk 1]
```diff
@@ -43,35 +42,55 @@ Configure Docker to use Kata Containers\* by default
...
-You successfully installed and set up Kata Containers\* on |CLOSIA|.
+You have successfully installed and set up Kata Containers on |CLOSIA|.
 
-More information about Docker\* in |CLOSIA|.
+More information about Docker in |CLOSIA|.
 #############################################
 
-Docker\* on |CLOSIA| provides a docker.service service file to start the Docker\*
+Docker on |CLOSIA| provides a docker.service service file to start the Docker
 daemon. The daemon will use runc or cc-runtime depending on the environment:
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: s/cc-runtime/kata-runtime

File: source/clear-linux/tutorials/kata.rst [hunk 2]
```diff
@@ -43,35 +42,55 @@ Configure Docker to use Kata Containers\* by default
...
 
 If you are running |CL| on baremetal or on a VM with Nested
-Virtualization activated, Docker\* will use cc-runtime as the default runtime.
-If you are running |CL| on a VM without Nested Virtualization, Docker\*
-will use runc as the default runtime. It is not necessary to configure Docker\*
-to use cc-runtime manually since Docker\* itself will automatically use this
+Virtualization activated, Docker will use cc-runtime as the default runtime.
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: s/cc-runtime/kata-runtime

File: source/clear-linux/tutorials/kata.rst [hunk 3]
```diff
@@ -43,35 +42,55 @@ Configure Docker to use Kata Containers\* by default
...
+If you are running |CL| on a VM without Nested Virtualization, Docker
+will use runc as the default runtime. It is not necessary to configure Docker
+to use cc-runtime manually since Docker itself will automatically use this
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: s/cc-runtime/kata-runtime

File: source/clear-linux/tutorials/kata.rst [hunk 4]
```diff
@@ -43,35 +42,55 @@ Configure Docker to use Kata Containers\* by default
...
    sudo systemctl daemon-reload
    sudo systemctl restart docker
 
-Run Kata Containers\*
+Run Kata Containers
 *******************
 
 .. code-block:: bash
 
    sudo docker run -ti busybox sh
```
Comment (tpleavitt) [inline source/clear-linux/tutorials/kata.rst]: When I run this command, after following the note below to set up the proxy variables, I get the following error:

Unable to find image 'busybox.sh:latest' locally
docker: Error response from daemon: pull access denied for busybox.sh, repository does not exist or may require 'docker login'.
See 'docker run --help'.

So I'm making progress but I wasn't expecting the requirement to login to docker.  running the 'docker run --help' does not show me an option to login.  There is a 'docker login' option.  If a end user runs into this we need to walk them through it

Comment (tpleavitt) [reply source/clear-linux/tutorials/kata.rst]: running 'docker login' generates the following:
Warning: failed to get default registry endpoint from daemon (Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.37/info: dial unix /var/run/docker.sock: connect: permission denied). Using system default: https://index.docker.io/v1/
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.

Comment (tpleavitt) [reply source/clear-linux/tutorials/kata.rst]: running 'sudo docker login' removes the warning and displays only the last line:
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.

Comment (tpleavitt) [reply source/clear-linux/tutorials/kata.rst]: Issue resolved!

**Comment by mvincerx** [comment]
Thanks all!

File: source/clear-linux/tutorials/kata.rst [hunk 5]
```diff
@@ -43,36 +42,57 @@ Configure Docker to use Kata Containers\* by default
...
 
-If you are running |CL| on baremetal or on a VM with Nested
-Virtualization activated, Docker\* will use cc-runtime as the default runtime.
-If you are running |CL| on a VM without Nested Virtualization, Docker\*
-will use runc as the default runtime. It is not necessary to configure Docker\*
-to use cc-runtime manually since Docker\* itself will automatically use this
-runtime on systems that support it.
+If you are running |CL| on bare metal or on a VM with Nested
+Virtualization activated, Docker will use s/cc-runtime/kata-runtime as the
+default runtime. If you are running |CL| on a VM without Nested
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: I'm sorry, `s/cc-runtime/kata-runtime` means to replace cc-runtime with kata-runtime, and I think you didn't understand it, sorry for the misunderstood 

please remove `s/cc-runtime`

`Virtualization activated, Docker will use kata-runtime as the`

File: source/clear-linux/tutorials/kata.rst [hunk 6]
```diff
@@ -43,36 +42,57 @@ Configure Docker to use Kata Containers\* by default
...
+You have successfully installed and set up Kata Containers on |CLOSIA|.
 
-More information about Docker\* in |CLOSIA|.
+More information about Docker in |CLOSIA|.
 #############################################
 
-Docker\* on |CLOSIA| provides a docker.service service file to start the Docker\*
-daemon. The daemon will use runc or cc-runtime depending on the environment:
+Docker on |CLOSIA| provides a docker.service service file to start the Docker
+daemon. The daemon will use runc or s/cc-runtime/kata-runtime depending on
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: please remove 's/cc-runtime'

**Comment by mvincerx** [comment]
Hello @devimc. I will make these changes in a new PR in which I will select you as the reviewer.

**Comment by devimc** [comment]
@mvincerx thanks

File: source/clear-linux/tutorials/kata.rst (hunk 5 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/kata.rst]: PR 192 resolves this. 
https://github.com/clearlinux/clear-linux-documentation/pull/192

**Comment by mvincerx** [comment]
PR 192 resolves this. 
https://github.com/clearlinux/clear-linux-documentation/pull/192

**Comment by mythi** [comment]
kata.rst and kata_migration.rst should be updated further to adapt to the latest changes: the runtime selection can be done via ```$DOCKER_DEFAULT_RUNTIME``` systemd Environment.

**Comment by mvincerx** [comment]
@mythi Thank you for your input.

**Comment by MCamp859** [comment]
@mvincerx  I am working with @mythi to update the rst files, please stay tuned...

**Comment by mvincerx** [comment]
@MCamp859. Thank you very much!

**Pull request merged**